### DISCUSSION TITLE
Docker version has stopped working

### DISCUSSION THREAD

**Discussion by bigone72** [mat/besticon#83] [Q&A]
About a year ago I installed the docker version, working perfectly. But for a while, I don't know exactly when, the service doesn't return any icon.

From the web, the result is always: "this poor site has no icons at all :-("
If I use "allicons", the result is always: {"url":"domain","icons":[]}

I think the last version in docker is from two years ago, instead, the last update in GitHub is from March.

Could you update the docker version? Or could you guide me to solve the problem?

Thank you.

**Comment by mat** [comment]
Unfortunately I cannot reproduce this, it's working for me. However, I just released a new version of the Docker image https://hub.docker.com/r/matthiasluedtke/iconserver/tags Maybe you can give it a try.
